### PR TITLE
feat(tx-history): add adjustable tx history page size 

### PR DESCRIPTION
**Pull request by amateima**
Screenshot 2022-05-16 at 20 52 54

### CODE REVIEW

File: src/views/Transactions/useTransactionsView.ts [hunk 1]
```diff
@@ -76,6 +82,11 @@ export default function useTransactionsView() {
     };
   }, [timer]);
 
+  function onPageSizeChange(value: number) {
+    setPageSize(value);
+    localStorage.setItem(TX_HISTORY_PAGE_SIZE_KEY, value.toString());
+  }
+
```
Comment (amateima) [inline src/views/Transactions/useTransactionsView.ts:89]: Cache the user preference in terms of page size

Comment (Tulun) [reply src/views/Transactions/useTransactionsView.ts:89]: I might split this logic into it's own utility function. WYDT?

**Review by Tulun — COMMENTED** [review]
One small comment. But this is basically just a port from the OO UI?

**Comment by amateima** [comment]
> One small comment. But this is basically just a port from the OO UI?

Actually I didn't take a look there to see how it was implemented

File: src/views/Transactions/useTransactionsView.ts (hunk 1 above)
Comment (amateima) [reply src/views/Transactions/useTransactionsView.ts]: Sure. [Changes here](https://github.com/across-protocol/frontend-v2/pull/89/commits/73de817757abeded16343066dff43ad4d53052bf)

**Review by Tulun — APPROVED** [review]

**Review by Tulun — APPROVED** [review]

**Review by daywiss — COMMENTED** [review]
code looks good, works well, but you also have to apply this to the mobile table

**Review by daywiss — APPROVED** [review]

**Pull request merged**